azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -86,7 +105,7 @@ stages:
               azureSubscription: "$(AZM_SERVICE_CONNECTION)"
               resourceGroupName: "$(ENVIRONMENT)-rg-$(AZM_FUNCTION_NAME)-$(AZM_LOCATION)"
               csmFile: 'azuredeploy.json'
-              overrideParameters: "-location $(AZM_LOCATION) -environment $(ENVIRONMENT) -functionAppName $(AZM_FUNCTION_NAME) -inboundAllowedIps $(INBOUND_ALLOWED_IPS)"
+              overrideParameters: '-location $(AZM_LOCATION) -environment $(ENVIRONMENT) -functionAppName $(AZM_FUNCTION_NAME) -inboundAllowedIps $(INBOUND_ALLOWED_IPS) -oldOutboundIpAddresses $(outboundIPAddresses)'
```

Comment:
Rollback this line, please.

Reply:
We need to have this parameter to pass the list of outbound IP addresses to the ARM. I changed the quotes to doble.